docs/01-overview/02-main-areas/service-management/smgt-05-sc-deprovisioning.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Unbinding and deprovisioning flow
+---
+
+Unbinding and deprovisioning are processes reverse to provisioning and binding. The diagram shows an overview of interactions between all resources related to the processes of unbinding and deprovisioning a service:
+
+![Unbinding and deprovisioning](./assets/unbinding-deprovisioning.svg)
+
+## Unbind a service
+
+You can unbind a service in two ways:
+- Delete a ServiceBindingUsage. During this process, the Secret injection is deleted, but the Secret itself still exists in the Namespace.
+- Delete a ServiceBinding. It deletes the Secret and triggers the deletion of all related ServiceBindingUsage custom resources.
+
+![Unbinding](./assets/unbinding.svg)
+
+## Deprovision a service
```

Comment:
same here, it's somehow instructional rather than explanatory. However, not sure how many steps that'll have?

Reply:
It will be a very short tutorial, but let's go with this approach 🙂